lib/index.js

```diff
@@ -90,6 +118,13 @@ function createHttpClient(options) {
 }
 
 
+/**
+ * creates a server.
+ * @public
+ * @function createServer
+ * @param    {Object} options an options object
```

Comment:
as a TODO, once the new docs are in place, we should link to them from within the jsdocs.